Recipes/Mains-Advanced/Neutral_density.ipynb

```diff
@@ -9,9 +9,11 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        bathymetry = catalog[experiment].search(variable = bathymetry_variable, path=".*output000.*").to_dask()[bathymetry_variable]    
</pre><p>Not for action for this pull request, but I'd love to figure out how we could do this without relying on searching on <code>path</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/536/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>